doc/source/user/usage.rst

```diff
@@ -366,3 +372,62 @@ As with the :meth:`~watts.Database.remove` method, clearing the database will
 delete all the corresponding results on disk, including copies of the input and
 output files from the workflow stored in the data directory. Original input
 files stored outside the database directory will be unaffected.
+
+Command-line Tool
+~~~~~~~~~~~~~~~~~
+
+The ``watts`` command-line tool provides an easy way to inspect results stored
+in the database. This tool has three subcommands:
+
+results
+    Displays a list of all results in the database
+dir
+    Provides the directory for a specific result (referenced by index)
+stdout
+    Shows the standard output from a specific result (referenced by index)
+
+The ``results`` subcommand will produce a table such as the following:
+
+.. code-block:: console
+
+    $ watts results
+    +-------+--------+--------+--------+----------------------------+
+    | Index | Job ID | Plugin | Name   | Time                       |
+    +-------+--------+--------+--------+----------------------------+
+    | 0     | 1      | MCNP   |        | 2022-06-01 13:21:44.713942 |
+    | 1     | 2      | MCNP   |        | 2022-06-01 13:23:12.410774 |
+    | 2     | 3      | MCNP   | r=2.0  | 2022-06-02 07:46:05.463723 |
+    | 3     | 3      | MCNP   | r=4.0  | 2022-06-02 07:46:10.996932 |
+    | 4     | 3      | MCNP   | r=6.0  | 2022-06-02 07:46:17.487411 |
+    | 5     | 3      | MCNP   | r=8.0  | 2022-06-02 07:46:24.964455 |
+    | 6     | 3      | MCNP   | r=10.0 | 2022-06-02 07:46:33.426781 |
+    +-------+--------+--------+--------+----------------------------+
```

Comment:
same here - we need to update Job ID.
Also, can you add a short description of the `watts rm Index` command?